src/main/java/com/github/igotyou/FactoryMod/ConfigParser.java

```diff
@@ -810,6 +814,25 @@ private IRecipe parseRecipe(ConfigurationSection config) {
 		case "DUMMY":
 			result = new DummyParsingRecipe(identifier, name, productionTime, null);
 			break;
+		case "PRINTINGPLATE":
+			ItemMap printingPlateInput = parseItemMap(config.getConfigurationSection("input"));
```

Comment:
Ok, thanks for suggestion.
I've used more earlier version of FactoryMod when started development where all of this seems was absent.